CHANGELOG.md

```diff
@@ -6,8 +6,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.9.0
+
  - Added AsyncMultipartUpload.
  - Add S3Object.
+ - Updated `aws_lambda_events` to 0.7.2 
+ - Updated `aws_lambda_runtime` to 0.7.1
+ - Updated `clap` to 4.0.27 
+ - Updated `reqwest` to 0.11.13 
+ - Updated `serde_json` to 1.0.89 
+ - Updated `tokio` to 1.22.0 
+ - Updated `serde` to 1.0.147 
+ - Updated `anyhow` to 1.0.66 
+ - Updated `tracing-subscriber` to 0.3.16 
+ - Updated `tracing` to 0.1.37
+ - Updated `async-trait` to 0.1.58 
+ - Updated `futures` to 0.3.25 
```

Comment:
In general in previous releases we've only listed the `aws-*` packages, and then any additional significant changes (e.g. when we moved to `clap` `4.x.x`).

```suggestion
```